lib/ansible/modules/tempfile.py

```diff
@@ -66,6 +66,12 @@
     state: file
     suffix: temp
   register: tempfile_1
+  
```

Comment:
ERROR: lib/ansible/modules/tempfile.py:69:1: W293: blank line contains whitespace (100%)

Reply:
@Akasurde

- Sorry, initially dint understand what you meant. 
- Removed the white spaces and pushed the changes for your review.